MovieGPT-api/src/main/java/lt/techin/group/project/model/Genre.java

```diff
@@ -19,7 +18,6 @@ public class Genre {
     @Id
     @GeneratedValue
     private Long id;
-    @Column(unique = true)
     private String name;
 
```

Comment:
Remove Data annotation. 
  
   Write this and now we'll have same genre with many movies database table.
```
 @ManyToMany(fetch = FetchType.EAGER)
    private Set<Media> medias;
```

Reply:
Solved, used Getter and Setter instead, ManyToMany introduced (wasnt working before due to Lombok Data annotation)